app/kubemci/pkg/gcp/backendservice/backendservicesyncer.go

```diff
@@ -114,16 +116,34 @@ func (b *BackendServiceSyncer) ensureBackendService(lbName string, port ingressb
 	existingBE, err := b.bsp.GetGlobalBackendService(name)
 	if err == nil {
 		fmt.Println("Backend service", name, "exists already. Checking if it matches our desired backend service")
-		glog.V(5).Infof("Existing backend service: %v\n, desired backend service: %v", existingBE, *desiredBE)
+		jsonExisting, mErr := json.Marshal(existingBE)
+		if mErr != nil {
+			glog.Infof("Error marshaling existing backend: %v", mErr)
+		}
+		jsonDesired, mErr := json.Marshal(desiredBE)
+		if mErr != nil {
+			glog.Infof("Error marshalling dsired backend: %v", mErr)
+		}
+		glog.V(5).Infof("Existing backend service:\n%s\nDesired backend service:\n%s", jsonExisting, jsonDesired)
 		// Backend service with that name exists already. Check if it matches what we want.
 		if backendServiceMatches(desiredBE, existingBE) {
 			// Nothing to do. Desired backend service exists already.
-			fmt.Println("Desired backend service exists already")
+			fmt.Println("Desired backend service exists already. Nothing to do.")
 			return existingBE, nil
 		}
 		// TODO (nikhiljindal): Require explicit permission from user before doing this.
-		fmt.Println("Updating existing backend service", name, "to match the desired state")
-		return b.updateBackendService(desiredBE)
+		if forceUpdate {
+			// TODO(G-Harmon): Figure out how to properly calculate the fp. Using Sha256 returned a googleapi error.
+			desiredBE.Fingerprint = existingBE.Fingerprint
+			fmt.Println("Updating existing backend service", name, "to match the desired state (since --force specified)")
+			return b.updateBackendService(desiredBE)
+		} else {
+			// TODO(G-Harmon): Show diff to user and prompt yes/no for overwriting.
+			fmt.Println("Will not overwrite a differing BackendService without the --force flag.")
```

Comment:
Could you extract the logic above that shows JSON versions into a method and call that here too?

(not necessary now)